pkg/windows/windows.go

```diff
@@ -547,18 +535,18 @@ func (vm *windows) ConfigureAzureCloudNodeManager(nodeName string) error {
 }
 
 func (vm *windows) ConfigureKubeProxy(nodeName, hostSubnet string) error {
-	endpointIP, err := vm.createHNSEndpoint()
+	endpointIP, err := vm.Run(NetworkConfScriptPath, true)
```

Comment:
Why are you combining CNI configuration and kube proxy configuration?

Reply:
@aravindhp This is happening because I wanted to limit the amount of churn that would occur between commits 2 and 3. There is no real "kube proxy configuration" that takes place, besides starting the kube-proxy service. The powershell commands that are run are still setting up the CNI by creating a usable endpoint for the HNS network.
Should I squash those commits together?

Reply:
Did you create a follow up story to separate this out i.e. introduce allowing WICD to run PowerShell scripts.

Reply:
Yes, it was added as a TODO in the code. https://issues.redhat.com/browse/WINC-882